### PR TITLE
Snowbridge - Disable create agent and create channel calls

### PR DESCRIPTION
**Pull request by claravanstaden**
Disable the Snowbridge create agent and channel calls, since they are not used and will be overhauled in v2.

### CODE REVIEW

**Comment by claravanstaden** [comment]
@alistair-singh @vgeddes @yrong ready for review.

**Review by acatangiu — APPROVED** [review]

**Review by vgeddes — APPROVED** [review]

**Review by alistair-singh — APPROVED** [review]

**Comment by acatangiu** [comment]
@claravanstaden please fix clippy

@joepetrowski or @muharem this needs another rank2 review

File: system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs [hunk 1]
```diff
@@ -226,6 +226,23 @@ parameter_types! {
 	pub const SS58Prefix: u8 = 0;
 }
 
+pub struct BaseFilter;
```
Comment (joepetrowski) [inline system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs:229]: `pub` item should have docs.

File: system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs [hunk 2]
```diff
@@ -226,6 +226,23 @@ parameter_types! {
...
+impl Contains<RuntimeCall> for BaseFilter {
+	fn contains(call: &RuntimeCall) -> bool {
+		// Disallow these Snowbridge system calls.
+		if matches!(
+			call,
+			RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_agent { .. })
+		) || matches!(
+			call,
+			RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_channel { .. })
+		) {
```
Comment (joepetrowski) [inline system-parachains/bridge-hubs/bridge-hub-polkadot/src/lib.rs:239]: Just personal preference, this looks nicer.
```suggestion
			RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_agent { .. }) |
				RuntimeCall::EthereumSystem(snowbridge_pallet_system::Call::create_channel { .. })
		) {
```

**Review by joepetrowski — APPROVED** [review]

**Pull request merged**